xc7/utils/prjxray_constant_site_pins.py

```diff
@@ -0,0 +1,66 @@
+""" This file defines tieable wires in 7-series designs.
+
+FEATURES_WHEN_ROUTED is a map of wires connected to site pins to a FASM feature.
+If the wire is routed from the general routing network, then this feature must
+also be emitted.
+
+WIRE_TIE_OPTIONS is a map of wires connected to site pins to the list of tie
+options available for the pin.
+
+"""
+
+
+# Using these wires requires a mux to be enabled.
+FEATURES_WHEN_ROUTED = {
```

Comment:
I'm not sure these `XXUSEDMUX` are correct.

Shouldn't these just be the following inside the `<pb_type>`;
```xml
<interconnect>
<mux name="SRUSEDMUX" input="BLK_IG-SLICE_X1.CE" output="SLICE.CE" >
  <metadata>
    <meta name="fasm_mux">
      COMMON_SLICE_X1.CE : SRUSEDMUX
    </meta>
  </metadata>
</mux>
</interconnect>
```

Reply:
That doesn't allow support for tying of SR and CE to VCC.  What you have is what there was before, which requires a routed VCC rather than the HARD1 inside of the site.

Reply:
It does make me wonder whether a techmap to disconnect SR and CE might work better.

Reply:
Removed the XXUSEDMUX changes in light of found complexity.  Will take another pass at them in after merging the other changes.